### PR TITLE
Tdr 3026 update standard faqs

### PR DESCRIPTION
**Pull request by TomJKing**

### CODE REVIEW

File: app/views/faq.scala.html [hunk 1]
```diff
@@ -123,46 +127,47 @@ <h3 class="govuk-heading-s">Can I share my sign-in information?</h3>
...
             </p>
 
-            <h3 class="govuk-heading-s">How do I know if my records are Crown copyright?</h3>
+            <h3 class="govuk-heading-s">How do I know if my records are Crown Copyright?</h3>
             <p class="govuk-body">
-                Works created by officers or servants of the crown during the course of their duties are Crown copyright. This includes material created by ministers, civil servants, government departments and agencies. The National Archives will accept Public Records that are not Crown copyright, however during this initial private beta stage, you cannot currently use TDR to transfer them to us.
+                Works created by officers or servants of the crown during the course of their duties are Crown Copyright. This includes material created by ministers, civil servants, government departments and agencies. In the future The National Archives will accept Public Records that are not Crown Copyright, however during this initial development stage, you cannot use TDR to transfer them to us.
```
Comment (MancunianSam) [inline app/views/faq.scala.html:151]: The word document has got "officers or servants of the **C**rown" if I'm reading it right and copyright is all lower case in the heading and paragraph.

File: app/views/faq.scala.html [hunk 2]
```diff
@@ -283,42 +293,78 @@ <h3 id="metadata-captured" class="govuk-heading-s">What metadata will be capture
...
                 @Messages("nationalArchives.email")</a>.
             </p>
 
+            <h3 class="govuk-heading-s">What do I do if the title or description of my record is sensitive to the public?</h3>
+            <p class="govuk-body">
+                If the title or description of your record is sensitive to the public, you can provide an alternative when adding closure metadata. The alternative will be visible on the public catalogue. You may wish to do one of the following:
+            </p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>Redact - remove the sensitive words. For example: 'Correspondence with [name withheld]'.</li>
+                <li>Sanitise - replace the sensitive word. For example: 'UK military exercises in xx' (where 'military' has replaced 'Special Forces').</li>
```
Comment (MancunianSam) [inline app/views/faq.scala.html:306]: The single quotes on this line don't exactly match the single quotes in the document, although I think this way looks better.

File: app/views/faq.scala.html [hunk 3]
```diff
@@ -283,42 +293,78 @@ <h3 id="metadata-captured" class="govuk-heading-s">What metadata will be capture
...
+                <li>Withhold the title or description without providing an alternative. For example: '[Title withheld]' and or '[description withheld]'</li>
+            </ul>
+            <p class="govuk-body"></p>
+            <p class="govuk-body"></p>
+            <p class="govuk-body"></p>
```
Comment (MancunianSam) [inline app/views/faq.scala.html:311]: What are these empty paragraph tags for?

**Review by MancunianSam — COMMENTED** [review]
Looks good, thanks. Just a couple of comments.

File: app/views/faq.scala.html (hunk 2 above)
Comment (TomJKing) [reply app/views/faq.scala.html:306]: Bit of editorial licence, as the quotes in the doc didn't make sense as they were.

File: app/views/faq.scala.html (hunk 3 above)
Comment (TomJKing) [reply app/views/faq.scala.html]: A reminder to delete them ;)

File: app/views/faq.scala.html (hunk 1 above)
Comment (TomJKing) [reply app/views/faq.scala.html]: There was inconsistencies with the capitalisation of the "Crown" and "Copyright", so its consistent now.

**Review by MancunianSam — APPROVED** [review]
Looks good, I'm happy if @thanhz is happy.

**Review by thanhz — APPROVED** [review]

**Pull request merged**